pkg/util/constants/constant_darwins.go

```diff
@@ -0,0 +1,6 @@
+package constants
```

Comment:
File name typo, should be `constant_darwin.go`, not `constant_darwins.go`.

Likely the cause of the lint failure below

Reply:
Name changed